Bug with building docs - Readme is not correct

I was just trying to follow the [Readme.md](https://github.com/projectmesa/mesa/blob/master/docs/README.md) to build the docs, and I get the following error: 

```
mesa/docs [git fix-docs] $ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.3.6

Recursion error:
maximum recursion depth exceeded while calling a Python object

This can happen with very large or deeply nested source files.  You can carefully increase the default Python recursion limit of 1000 in conf.py with e.g.:
    import sys; sys.setrecursionlimit(1500)
make: *** [html] Error 1
```

Not sure why I am running into this. I feel like I have come across this before, but I can't remember how I fixed.

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -17,19 +17,6 @@ import sys
 import os
 
 
-# Adding mock imports to see if this builds
-from unittest.mock import MagicMock
-
-class Mock(MagicMock):
-    @classmethod
-    def __getattr__(cls, name):
-            return Mock()
-
-MOCK_MODULES = ['numpy', 'pandas']
-sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
-
-# End of mock
-
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.

